pkg/features/kube_features.go

```diff
@@ -330,6 +330,18 @@ const (
 	// Disables the OpenStack Cinder in-tree driver.
 	InTreePluginOpenStackUnregister featuregate.Feature = "InTreePluginOpenStackUnregister"
 
+	// owner: @humblec
+	// alpha: v1.23
+	//
+	// Enables the RBD in-tree driver to RBD CSI Driver  migration feature.
+	CSIMigrationRbd featuregate.Feature = "CSIMigrationRbd"
```

Comment:
`CSIMigrationRBD`
(abbreviations have all the same case, like CSI)

Reply:
sure, done